content/version/2/0/code_of_conduct.pt-br.md

```diff
@@ -0,0 +1,90 @@
++++
+version = "2.0"
+aliases = ["/version/2/0"]
++++
+
+# Código de Conduta para Colaboradores
+
+## Nossa promessa
+
+Como participantes, colaboradores e lideranças, nós nos comprometemos a fazer com que a participação em nossa comunidade seja uma experiência livre de assédio para todas as pessoas, independentemente de idade, tamanho do corpo, deficiência aparente ou não aparente, etnia, características sexuais, identidade ou expressão de gênero, nível de experiência, educação, situação sócio-econômica, nacionalidade, aparência pessoal, raça, religião ou identidade e orientação sexuais.
```

Comment:
I saw that you changed "liderenças" to "liderança" in multiple places. Shouldn't you change it here too? 
```suggestion
Como participantes, colaboradores e liderança, nós nos comprometemos a fazer com que a participação em nossa comunidade seja uma experiência livre de assédio para todas as pessoas, independentemente de idade, tamanho do corpo, deficiência aparente ou não aparente, etnia, características sexuais, identidade ou expressão de gênero, nível de experiência, educação, situação sócio-econômica, nacionalidade, aparência pessoal, raça, religião ou identidade e orientação sexuais.
```

Reply:
In this particular case I left it intentionally, because it sounded weird with "nós" coming right after "liderança". But, come to think of it, and as someone else pointed out before, "liderança" could also mean a collective of leaders. So, for standards sake, I'm changing it to singular here too.